packages/components/menu/src/menu-button.tsx

```diff
@@ -95,6 +95,9 @@ export const MenuButton = forwardRef<MenuButtonProps, "button">(
   },
 )
 
+MenuButton.displayName = "MenuButton"
+MenuButton.__ui__ = "MenuButton"
+
 const Button = forwardRef<MenuButtonProps, "button">((rest, ref) => {
   const { styles } = useMenu()
 
```

Comment:
It looks like Button is missing displayName and `__ui__`.
Since it uses forwardRef, Button.displayName = "Button" and Button.__ui__ = "Button" should be added to keep it consistent with other components.

Reply:
I heard from @hirotomoyamada that they are unnecessary in this case.

Reply:
Sorry, That's OK.